apigee_m10n.routing.yml

```diff
@@ -35,17 +55,56 @@ apigee_monetization.packages:
   requirements:
     _permission: 'access monetization packages'
 
-apigee_monetization.my_purchased:
-  path: /user/monetization/packages/purchased
+apigee_monetization.my_subscriptions:
+  path: /user/monetization/subscriptions
+  defaults:
+    _controller: \Drupal\apigee_m10n\Controller\PackagesController::mySubscriptions
+  requirements:
+    _permission: 'view subscription'
```

Comment:
These permission feel really generic, so generic that  they might collide with something provided by commerce